storage/persistent-storage/persistent-storage-local.adoc

```diff
@@ -0,0 +1,29 @@
+[id="persistent-storage-using-local-volume"]
+= Persistent storage using local volumes
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-local
+
+toc::[]
+
+{product-title} can be provisioned with persistent storage by using
+local volumes. Local persistent volumes allow you to access local storage
+devices such as a disk, partition, or directory by using the standard
+PVC interface.
```

Comment:
actually, we do not support consuming plain directories as local-volumes right now. We should drop that from docs IMO.